mithril-aggregator/src/http_server/routes/signatures_routes.rs

```diff
@@ -40,9 +39,10 @@ mod handlers {
         unwrap_to_internal_server_error, MetricsService, SingleSignatureAuthenticator,
     };
 
+    const HTTP_ORIGIN: &str = "HTTP";
```

Comment:
A more specific name could avoid confusions
```suggestion
    const METRICS_HTTP_ORIGIN: &str = "HTTP";
```